examples/README.md

```diff
@@ -2,13 +2,18 @@
 
 The `ballerinax/openai.audio` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-openai.audio/tree/main/examples), covering use cases like cache management, session management, and rate limiting.
 
-[//]: # (TODO: Add examples)
-1. 
-2. 
+1. [Text-To-English-Audio](https://github.com/ballerina-platform/module-ballerinax-openai.audio/tree/main/examples/Text-To-English-Audio) - Converts a text given in any language to english
+2. [Audio-To-Input-Language-And-English](https://github.com/ballerina-platform/module-ballerinax-openai.audio/tree/main/examples/Audio-To-English-Audio) - Converts an audio given in a different language into text in input language and english
```

Comment:
Need to update the examples section in the repo README.md, Module.md and Package.md as well